src/sentry/grouping/grouptype.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import dataclass
+
+from sentry.issues.grouptype import GroupCategory, GroupType
+from sentry.models.group import DEFAULT_TYPE_ID
+from sentry.types.group import PriorityLevel
+from sentry.workflow_engine.endpoints.validators.error_detector import ErrorDetectorValidator
+
+
+@dataclass(frozen=True)
+class ErrorGroupType(GroupType):
+    type_id = DEFAULT_TYPE_ID
+    slug = "error"
+    description = "Error"
+    category = GroupCategory.ERROR.value
+    default_priority = PriorityLevel.MEDIUM
+    released = True
+    detector_validator = ErrorDetectorValidator
```

Comment:
for the future - do we want to add a link to a detector handler that includes the project settings stuff here? that could be a pretty neat way to integrate detectors into grouping rules.

Reply:
yes i can add it if we need!